org.gnome.chess.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.gnome.chess",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "gnome-chess",
+    "copy-icon": true,
+    "rename-icon": "gnome-chess",
+    "rename-desktop-file": "gnome-chess.desktop",
+    "rename-appdata-file": "gnome-chess.appdata.xml",
```

Comment:
Obviously the best would be to have these renames done upstream.

Reply:
Yes, but till then I will do it in flatpak

Reply:
The changes would certainly be welcome upstream, just remember to update the hardcoded list of renamed desktop files in gnome-shell at the same time...